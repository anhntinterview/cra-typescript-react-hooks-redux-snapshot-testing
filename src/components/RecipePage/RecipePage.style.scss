.recipe-wrapper {
  width: 40vw;
  height: 20vw;
  display: flex;
  overflow-y: scroll;
  overflow-x: hidden;
  flex-direction: column;
  padding: 1.5rem;
  position: relative;
  margin-top: -10rem;
}

@media (max-width: 768px) {
  .recipe-wrapper {
    width: 90vw;
    height: 100vw;
  }
}


