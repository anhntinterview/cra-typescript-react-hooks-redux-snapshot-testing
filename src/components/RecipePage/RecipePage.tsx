import * as React from "react";

import Filter from "./Filter";
import TheList from "./TheList";

import "./RecipePage.style.scss";

export interface RecipePageProps {}

const RecipePage: React.FC<RecipePageProps> = () => {
  return (
    <div className="recipe-wrapper">
      <Filter />
      <TheList />
    </div>
  );
};

export default RecipePage;
