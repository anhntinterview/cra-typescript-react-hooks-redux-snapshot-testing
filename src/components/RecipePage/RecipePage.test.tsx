import * as React from "react";
import { 
  shallow, 
} from "enzyme";

import RecipePage from "./RecipePage";

describe('RecipePage', () => {
  it('should match a snapshot', () => {
    expect(shallow(<RecipePage />)).toMatchSnapshot();
  })
})
