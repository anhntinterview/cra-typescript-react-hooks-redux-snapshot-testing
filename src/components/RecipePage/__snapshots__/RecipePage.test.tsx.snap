// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipePage should match a snapshot 1`] = `
<div
  className="recipe-wrapper"
>
  <Filter />
  <TheList />
</div>
`;
